src/System.Net.Http/src/System/Net/Http/Unix/HttpClientHandler.Unix.cs

```diff
@@ -39,8 +39,17 @@ public CookieContainer CookieContainer
 
         public ClientCertificateOption ClientCertificateOptions
         {
-            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
-            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            get
+            {
+                return ClientCertificateOption.Manual;
+            }
+            set
+            {
+                if (ClientCertificateOption.Manual != value)
+                {
+                    throw NotImplemented.ByDesignWithMessage("libcurl does not support automatic client certificate selection");
```

Comment:
Should this be PlatformNotSupportedException, since it sounds like libcurl doesn't support this? Or are we planning on implementing our own support?